src/app/components/dashboard/dashboard.component.html

```diff
@@ -69,6 +69,17 @@ <h5 class="modal-title" id="exampleModalLabel">Create Workspace</h5>
           <input type="text" class="form-control" aria-label="Default" aria-describedby="folder-description"
             id="folder-description-input">
         </div>
+        <!-- Folder Color -->
+        <div class = "input-group mb-3">
+          <div class = "input-group-prepend">
+            <span class = "input-group-text" id = "folder-colour">Color</span>
+          </div>
+            <input type="hidden" class="form-control" aria-label="Default" aria-describedby="folder-colour" value = "" id="folder-colour-input" readonly>
+            <button class = "folder_colour" style="background-color: #ee9ca7;" (click)="assignColour('#ee9ca7')"></button>
+            <button class = "folder_colour" style="background-color: #96e6a1;" (click)="assignColour('#96e6a1')"></button>
+            <button class = "folder_colour" style="background-color: #66a6ff;" (click)="assignColour('#66a6ff')"></button>
+            <button class = "folder_colour" style="background-color: #274046;" (click)="assignColour('#274046')"></button>
```

Comment:
Hey, 
Inline styling is not a good practice. Can you please create appropriate classes and attach

Thanks